### DISCUSSION TITLE
Table duplications in XMLA POwer BI service model

### DISCUSSION THREAD

**Discussion by 10xape** [TabularEditor/TabularEditor3#1402] [Data Modeling]
HI I have a Power BI service sementic model already uploaded , one of the tables recently started having some duplications in the relationship column. I was wondering if there is any way to run sort of Power Query duplication removal in TE# for that table without reuploding the entire model ? Thank you  Matt

**Comment by mlonsk** [comment]
Hi Matt (@10xape )

You would most likely need to create that duplicate removal inside your table Partition where the Power Query is stored. Tabular Editor does not currently have a way to do partial deployements, but you could utilize [ALM toolkit](http://alm-toolkit.com/) where you can select to only deploy the partition you have edited. 

However, as a general rule when duplicates happen I would consider first looking at the source system or in between ETL steps to identify where these are coming from and clean them out at the source.
